src/app/components/navbar/navbar.component.ts

```diff
@@ -4,7 +4,7 @@ import { SpotifyApiService } from 'src/app/services/spotify.service';
 @Component({
   selector: 'app-navbar',
```

Comment:
components like navbar should be placed in root component, but you've duplicated it 3 times